MessengerProj/src/main/res/values/strings.xml

```diff
@@ -413,7 +413,7 @@
     <string name="Never">Never</string>
     <string name="AutocryptKeyTransferInitiate">Send Autocrypt Setup Message</string>
     <string name="AutocryptKeyTransfer">Autocrypt Setup Message</string>
-    <string name="AutocryptKeyTransferMsgBeforeSend">An <![CDATA[<i>]]>Autocrypt Setup Message<![CDATA[</i>]]> securely shares your end-to-end setup with other Autocrypt-compliant devices.\n\nThe setup will be encrypted by a setup code which is displayed here and must be typed on the other device.</string>
+    <string name="AutocryptKeyTransferMsgBeforeSend">An <![CDATA[<i>]]>Autocrypt Setup Message<![CDATA[</i>]]> securely shares your end-to-end setup with other Autocrypt-compliant apps.\n\nThe setup will be encrypted by a setup code which is displayed here and must be typed on the other device.</string>
```

Comment:
it's the app, that is Autocrypt-compiant, not the device.

Reply:
Which is why I changed it to "app"  ┐(o.O)┌

Reply:
*sorry* :)